travis/test_server

```diff
@@ -239,13 +239,18 @@ def main():
             # Find errors, except from failed mails
             errors = has_test_errors(
                 "stdout.log", database, odoo_version, check_loaded)
+            if returncode != 0:
+                all_errors.append(to_test)
+                print(fail_msg, "Command exited with code %s" % returncode)
+                # If not exists errors then
+                # add an error when returcode!=0
+                # because is really a error.
+                if not errors:
+                    errors += 1
             if errors:
```

Comment:
I'm uneasy with moving the `if returncode != 0` upward.
Could a solution be to replace this  `if errors:` with `if errors or returncode != 0:`?